launch/launch/actions/execute_process.py

```diff
@@ -332,6 +332,10 @@ def __on_shutdown_process_event(
         self,
         context: LaunchContext
     ) -> Optional[LaunchDescription]:
+        typed_event = cast(ShutdownProcess, context.locals.event)
+        if not typed_event.process_matcher(self):
+            # this event whas not intended for this process
```

Comment:
Spelling.

Reply:
Also that `this` is used twice but references different entities is a bit hard to read.